File: senddb.java
import java.util.Scanner;
import java.sql.*;

/*
You can compile and run this example with a command like:
  javac senddb.java && 
  java -cp ~/bin/postgresql-42.0.0.jar:. senddb
You can download the postgres JDBC driver jar from https://jdbc.postgresql.org.
*/
public class senddb {
    public static void main(String[] args) throws ClassNotFoundException, SQLException {

        // Load the postgres JDBC driver.
        Class.forName("org.postgresql.Driver");

        // Connect to the database.
        Connection db = DriverManager.getConnection("jdbc:postgresql://127.0.0.1:26257/tpch?sslmode=disable", "root", "");

        try {
            Scanner input = new Scanner(System.in);
            while (input.hasNextLine()){
              db.createStatement().execute(input.nextLine());
            }
        } finally {
            // Close the database connection.
            db.close();
        }
    }
}

File: senddb.go
package main

import (
    "database/sql"
    "bufio"
    "os"
    "fmt"
    "log"

    _ "github.com/lib/pq"
)

func main() {

    // Connect to the database.
    db, err := sql.Open("postgres", "postgres://127.0.0.1:26257/tpch?sslmode=disable&user=root")
    if err != nil {
        log.Fatal("error connecting to the database: ", err)
    }

    scanner := bufio.NewScanner(os.Stdin)
    for scanner.Scan() {
        // Insert row(s) into the table(s).
        if _, err := db.Exec(scanner.Text()); err != nil {
            log.Fatal(err)
        }
    }
}

